Impossible-Agent-Suite/non-whitelist-mint/src/agent.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  Finding,
+  FindingType,
+  FindingSeverity,
+  TransactionEvent,
+  HandleTransaction,
+} from 'forta-agent';
+
+import {
+  Verifier,
+  IF_WHITELIST_VERIFIER,
+  IDIA_WHITELIST_VERIFIER,
+} from './verifier';
+
+// Address of the Impossible Finance token contract
+let IF_ADDR = '0xb0e1fc65c1a741b4662b813eb787d369b8614af1';
+let IDIA_ADDR = '0x0b15ddf19d47e6a86a56148fb4afffc6929bcb89';
+let ZERO_ADDR = '0x0000000000000000000000000000000000000000';
+
```

Comment:
```suggestion
// Address of the Impossible Finance token contract
const IF_ADDR = '0xb0e1fc65c1a741b4662b813eb787d369b8614af1';
const IDIA_ADDR = '0x0b15ddf19d47e6a86a56148fb4afffc6929bcb89';
const ZERO_ADDR = '0x0000000000000000000000000000000000000000';

```

Reply:
This has been changed in the latest commit.